### DISCUSSION TITLE
Copilot secure coding

### DISCUSSION THREAD

**Discussion by derrick-roach** [community/events#57] [AI]
### Does Copilot suggest _secure_ code by default? 

This would be a game-changer for developers. Remembering to sanitize inputs, prevent overflow, limit path traversal, and addressing other weaknesses is not usually top of mind when innovating. If developers can rely on Copilot to suggest secure code by default, then they can focus more on building. I would love to see/hear how the GitHub Copilot team is collaborating with the GitHub Advanced Security team to integrate secure coding into the AI pair programmer.

**Comment by Akash1134** [comment]
Hi @derrick-roach , Thanks for sharing your feedback. Our teams are continuously working together to make the AI ecosystem secure. We value your input and it makes us do more. Please feel free to share any specific feedback about Copilot security. You may also refer [Copilot FAQ's](https://github.com/features/copilot/#:~:text=Frequently%20asked%C2%A0questions) for more details.
